application/Profile/DBProfile.py

```
@@ -67,13 +73,17 @@
+        self.activation_cache = TTLCache(maxsize=MAX_PENDING_ACTIVATION_NUM, ttl=ACTIVATION_TTL_SECOND)
 
         # key: user_id, value: True (to indicate the entry exists; can be any dummy value)
-        self.resend_cooldown = TTLCache(maxsize=1024, ttl=RESEND_COOLDOWN_TTL_SECOND)
+        self.resend_cooldown = TTLCache(maxsize=MAX_PENDING_ACTIVATION_NUM, ttl=RESEND_COOLDOWN_TTL_SECOND)
 
-        with open('/var/www/ictrl/application/resources/activation_email_template.html', 'r') as f:
-            self.activation_email_body_template = f.read()
+        try:
+            with open(ACTIVATION_EMAIL_TEMPLATE) as f:
+                self.activation_email_body_template = f.read()
+        except OSError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)